cpp/ql/src/AlertSuppression.ql

```diff
@@ -64,7 +64,7 @@ class SuppressionScope extends ElementBase {
   * The location spans column `startcolumn` of line `startline` to
   * column `endcolumn` of line `endline` in file `filepath`.
   * For more information, see
-  * [LGTM locations](https://lgtm.com/help/ql/locations).
+  * [LGTM locations](https://help.semmle.com/QL/learn-ql/ql/locations.html).
```

Comment:
It would be a good idea to update the link text from "LGTM locations" too - it should probably be the title of the topic.